docs/program-setup/bare_metal/bare_metal.md

```diff
@@ -1,22 +1,19 @@
 # Mbed OS bare metal profile
 
-There are many use cases for IoT devices. Different use cases require different configurations, connectivity and security. They also have different requirements for resource consumption. Many products must operate in ultraconstrained environments on tiny MCUs with low memory and compute power available. We created the Mbed OS bare metal profile for IoT devices that require ultraconstrained resources.
+Bare metal is a profile of Mbed OS for ultraconstrained devices. Unlike the full Mbed OS, which by default includes all APIs, the bare metal profile starts with a minimal set of APIs to which you can add only the APIs your application or hardware demand. This helps you control the size of your final binary.<!--not sure that's a good term-->
 
-<span class="images">![Mbed OS bare metal profile block diagram](../../images/bare_metal_block_diagram.png)<span>Mbed OS bare metal profile block digram</span></span>
+Bare metal doesn't use the RTOS APIs. Instead, it relies on timers to control the workflow.
+<!--not sure I get it - I looked at both Blinky examples and they both use thread_sleep_for, even though the bare metal one doesn't include mbed_thread.h-->
+<!--what other APIs does it have or not have by default?-->
```

Comment:
Bare metal uses a subset of the RTOS APIs. These APIs have been ported to bare metal i.e. they do not make calls to RTX. By default, bare metal includes: drivers, platform and rtos-api https://github.com/ARMmbed/mbed-os/blob/master/platform/bare_metal/mbed_lib.json/#L3